.bit/responses/1.4-Week 1 Step 4.md

```diff
@@ -37,12 +37,12 @@ Azure is Microsoft's cloud computing platform (similar to Google Cloud Platform
 1. To create an Azure account, go to [Microsoft Azure](https://azure.microsoft.com/en-us/free/) and press **Start Free**.
 2. After signing in with your Microsoft account and filling in your personal details, you will be asked to add a credit card.
   - This is only for security purposes (preventing multiple free accounts per person).
-  - **You won't be charged** unless you choose to buy a premium account, which we do not need for this course. 
+  - **You won't be charged** unless you choose to buy a premium account, which we do not need for this course.
 
 ## Set Up Azure Locally on VS Code
 
 1. Confirm if you have installed the Azure Tools extension on VS Code
-2. [Create your local project](https://docs.microsoft.com/en-us/azure/azure-functions/create-first-function-vs-code-csharp#create-an-azure-functions-project) 
+2. [Create your local project](https://docs.microsoft.com/en-us/azure/azure-functions/create-first-function-vs-code-csharp#create-an-azure-functions-project)
```

Comment:
On step 6, I wouldn't say that you can get the function URL in the output of the terminal because my VScode doesn't give that to me. Maybe change it to so it says to get the function URL from Azure

Reply:
@ganning127 good point, also in this case it might be helpful to specify the difference between Function and Anonymous in #347. If you choose Anonymous, the URL outputs in the terminal.

Reply:
Updated will push now. Emily per your comment I thought we were suppose to be using anonymous functions in the curriculum.